source/_posts/2015-11-16-immutable-strings-in-ruby-2-dot-3.markdown

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: "Immutable strings in Ruby 2.3"
+category:
+date: 2015-11-16 08:39:25 -0300
+comments: true
+author:
+  name: Alexis Mas
+  email: alexis@wyeworks.com
+  twitter_handle: _axxiss
+  github_handle: axxiss
+  image: /images/team/
+  description: Software Engineer. Turning coffee into software since 2008.
+---
+
+String literals immutability is one of the new features you can find in [Ruby 2.3][1].
+But what does this implies?
+
+* Equality comparison affected
```

Comment:
I understand what you mean, but the operator is not affected what is changed is the fact that `The same literal strings are Identical` maybe better to write that?.
https://en.wikipedia.org/wiki/Identity_%28object-oriented_programming%29